go/src/lib.rs

```diff
@@ -1,9 +1,166 @@
-/**
+/*
  * Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
  */
-use std::ffi::c_void;
+#![deny(unsafe_op_in_unsafe_fn)]
```

Comment:
why prefer to deny the lint, instead of adding unsafe scopes inside the unsafe functions?

Reply:
This doesn't deny the lint, but actually enables it. See https://doc.rust-lang.org/beta/nightly-rustc/rustc_lint/builtin/static.UNSAFE_OP_IN_UNSAFE_FN.html

Reply:
my bad, missed that its a nightly lint, not a default one.